src/south/ProtocolHandler.class.js

```diff
@@ -174,6 +176,16 @@ class ProtocolHandler {
       setTimeout(resolve, timeout)
     })
   }
+
+  /**
+   * Get live status.
+   * @returns {object} - The live status
+   */
+  getStatus() {
```

Comment:
The default implementation should not be surcharged and could return standard variables. 

if we need a specific variable for this handler:

getStatus() {
const status = super.getStatus()
status.anykey = anyvalue
return status
}